kaldb/src/main/java/com/slack/kaldb/histogram/FixedIntervalHistogramImpl.java

```diff
@@ -56,8 +59,8 @@ public void add(long value) {
     if (value == high) {
       buckets.get(bucketCount - 1).increment(1);
     } else {
-      int bucketIdx = findMatchingBucket(buckets, 0, bucketCount, (double) value);
-      buckets.get(bucketIdx).increment(1);
+      int index = (int) Math.floor((value - low) / bucketSize);
```

Comment:
What happens when value is more than `high`?

Reply:
Good catch! Fixed and added an extra test.

Reply:
FYI I don't think this value will ever be higher than `high` in how we use it :) the collector I think will filter out matching docs in our time range before this gets invoked..